gix-config/src/file/access/comfort.rs

```diff
@@ -2,29 +2,29 @@ use std::{borrow::Cow, convert::TryFrom};
 
 use bstr::BStr;
 
-use crate::{file::MetadataFilter, value, File};
+use crate::{file::MetadataFilter, value, File, Key};
 
 /// Comfortable API for accessing values
 impl<'event> File<'event> {
     /// Like [`value()`][File::value()], but returning `None` if the string wasn't found.
     ///
     /// As strings perform no conversions, this will never fail.
-    pub fn string(
+    pub fn string_by(
         &self,
         section_name: impl AsRef<str>,
         subsection_name: Option<&BStr>,
         key: impl AsRef<str>,
     ) -> Option<Cow<'_, BStr>> {
-        self.string_filter(section_name, subsection_name, key, &mut |_| true)
+        self.string_filter_by(section_name, subsection_name, key, &mut |_| true)
```

Comment:
To reduce unnecessary monomorphprisation of generics:

```suggestion
        self.string_filter_by(section_name.as_ref(),  subsection_name, key.as_ref(), &mut |_| true)
```

Reply:
Fixed.